pyincore_viz/analysisviz.py

```diff
@@ -0,0 +1,26 @@
+import importlib
+
+
+class AnalysisViz:
+
+    @staticmethod
+    def visualize(dataset, **kwargs):
+        try:
+            module_name = ""
+            # split by namespace, capitalize then join
+            for item in dataset.data_type.split(":"):
+                module_name += item.capitalize()
+
+            # load module
+            # e.g. module_name = IncoreHousingUnitAllocation
+            module = importlib.import_module("pyincore_viz.analysis." + module_name.lower())
+
+            # load class
+            analysis_class = getattr(module, module_name)
+
+            # run vis
+            return analysis_class.visualize(dataset, **kwargs)
+
+        except:
+            raise ValueError("Fail to dynamically import dataset to its corresponding class. Please double "
+                             "check the data_type of the dataset!")
```

Comment:
oh im getting this error trying to run the corresponding doc PR 
![image](https://user-images.githubusercontent.com/13950475/136095543-0290db01-ffcb-4094-8dee-f492b5571ade.png)